src/centralstore/incremental/DataPublisher.cpp

```diff
@@ -0,0 +1,53 @@
+#include "./DataPublisher.h"
+
+#include "../../util/logger/Logger.h"
+#include "../../server/JasmineGraphInstanceProtocol.h"
+
+Logger data_publisher_logger;
+
+DataPublisher::DataPublisher(int worker_port, std::string worker_address) {
+    this->worker_port = worker_port;
+    this->worker_address = worker_address;
+    serv_addr.sin_family = AF_INET;
+    serv_addr.sin_port = htons(worker_port);
+    if (inet_pton(AF_INET, worker_address.c_str(), &serv_addr.sin_addr) <= 0) {
+        data_publisher_logger.error("Invalid address/ Address not supported!");
+    }
+}
+
+void DataPublisher::publish(std::string message) {
+    char recever_buffer[1024] = {0};
```

Comment:
Lets declare the 1024 as a constant. See for [example](https://github.com/miyurud/jasminegraph/blob/master/src/frontend/JasmineGraphFrontEndProtocol.h#L85).

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/122/commits/bdd734e88390169ddc9e44c74030d1c1b170d0ae

Reply:
Done